### DISCUSSION TITLE
How to do an outline of things to do or set objectives

### DISCUSSION THREAD

**Discussion by 2bitninja** [community/community#70072] [New to GitHub]
Hello I'm still very new to Git & GitHub.
What is the best practice for creating a to do list for your project? 

Example:
* Create script/s
* Add feature 1
* Add feature 2
* Create documentation
* Merge script 1 & script 2
* etc..

I looked at lists, but that looks more like bookmarks. 
If I create an issue, I'm asking for help. Which I may not always want.
I have not looked at projects yet.

Before I started to use git, I would create a little notebook file to track things. If I do that now, then it becomes part of the the git repo. It also gets included with my releases. I don't want that because the notebook file is not really part of the thing i'm creating.
Should I just track it in the README file? What do most people do?

**Comment by skadomsky** [comment]
1. Specifically, if you want to create a todo list just for yourself or your orgainzation, **'Project'** feature should be used then. It appears on the top bar of your repo interface, allowing you to create todo lists and assigned tasks to team member.

2. If you want to show what's next of your project to people who are interested in, then the **'Wiki'** feature is supposed. You can create pages of your todo lists, project documents, etc.

Both two ways mentioned above could help you create todo lists without file being tracked by Git. However, keeping a todo list in git repo doesn't appear to be such bad thing. A `TODO.md` file could help people understand more directly and could be edited anywhere.

Hope my answer could help you.

**Comment by 2bitninja** [comment]
Thanks for your help. I have been looking into GitHub Project and that seems to be the way to go. As for the putting the todo list in the repo. If I create a release then that file is also added. Unless there is a way to exclude files, well other than gitinore. It seems the list in project are issues in the repo.
